### DISCUSSION TITLE
Welcome to ttkbootstrap Discussions!

### DISCUSSION THREAD

**Discussion by israel-dryer** [israel-dryer/ttkbootstrap#59] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tcarnevale** [comment]
Just discovered the **_ttkbootstrap_** project.

**_Wow!  What a job you did!_**

I'm looking forward to playing with it.

Thanks!!

Tom Carnevale
